cmake/O2Dependencies.cmake

```diff
@@ -673,3 +673,37 @@ o2_define_bucket(
 
 )
 
+o2_define_bucket(
+    NAME
+    emcal_base_bucket
+
+    DEPENDENCIES
+    root_base_bucket
+    fairroot_base_bucket
+    ParBase
+    
+    INCLUDE_DIRECTORIES
+    ${FAIRROOT_INCLUDE_DIR}
+)
+
+o2_define_bucket(
+    NAME
+    emcal_simulation_bucket
+
+    DEPENDENCIES
+    emcal_base_bucket
+    root_base_bucket
+    fairroot_geom
+    RIO
+    Graf
```

Comment:
I would have at least RIO, Matrix, MathCore in the base bucket and then have a "root_plotting_bucket" with Graf.